jekyll/_cci2/github-integration.adoc

```diff
@@ -0,0 +1,279 @@
+
+For this reason, a deploy key isn't sufficiently powerful for projects with additional private dependencies.
+
+=== User key security
+
+CircleCI will never make your SSH keys public.
+
+The private keys of the checkout key-pairs CircleCI generates never leave the CircleCI systems (only the public key is transmitted to GitHub) and are safely encrypted in storage. However, since the keys are installed into your build containers, any code that you run in CircleCI can read them. Likewise, developers that can SSH in will have direct access to this key.
+
+Remember that SSH keys should be shared only with trusted users. GitHub collaborators on projects employing user keys can access your repositories, therefore, only entrust a user key to someone with whom you would entrust your source code.
+
+**Isn't there a difference between deploy keys and user keys?**
+
+Deploy keys and user keys are the only key types that GitHub supports. Deploy keys are globally unique (for example, no mechanism exists to make a deploy key with access to multiple repositories) and user keys have no notion of _scope_ separate from the user associated with them.
+
+To achieve fine-grained access to more than one repo, consider creating what GitHub calls a <<#controlling-access-via-a-machine-user,machine user>>. Give this user exactly the permissions your build requires, and then associate its user key with your project on CircleCI.
+
+=== User key access-related error messages
+
+Here are common errors that indicate you need to add a user key.
+
+**Python**: During the `pip install` step:
+
+```
+ERROR: Repository not found.
+```
+
+**Ruby**: During the `bundle install` step:
+
+```
+Permission denied (publickey).
+```
+
+== Add a .circleci/config.yml file
+
+After the necessary permissions have been set up, the next step is adding a `.circleci/config.yml` file to the projects you would like to use with CircleCI.
+
+Add a `.circleci` directory to a repository you want to connect to CircleCI. Inside that directory, add a `config.yml` file. You can use the basic configuration below inside your configuration file:
+
+```yaml
+version: 2.1
+
+orbs:
+    hello: circleci/hello-build@0.0.5
+
+workflows:
+    "Hello Workflow":
+        jobs:
+          - hello/hello-build
+
+```
+
+After you create and commit a `.circleci/config.yml` file to your GitHub repository, CircleCI immediately checks your code out and runs your first job along with any configured tests. This basic example runs a "hello world."
+
+CircleCI runs your tests on a clean container every time so that your tests are fresh each time you push code, and so that your code is never accessible to other users. Watch your tests update in real-time on https://circleci.com/dashboard[your dashboard]. You can also get status updates through email notifications, or look for the status badges that appear on GitHub. Integrated statuses also appear on the pull request screen, to show that all tests have passed.
```

Comment:
Instead of this mini guide to adding a config can we just push people here? https://circleci.com/docs/2.0/getting-started/ for this info? (one reason is this orb doesn't seem to exist anymore 😬 )